src/main/scala/com/typesafe/sbt/packager/rpm/Keys.scala

```diff
@@ -37,23 +37,32 @@ trait RpmKeys {
   val rpmConflicts = SettingKey[Seq[String]]("rpm-conflicts", "Packages this RPM conflicts with.")
   val rpmDependencies = SettingKey[RpmDependencies]("rpm-dependencies", "Configuration of dependency info for this RPM.")
 
+  // MAINTAINER SCRIPTS
+  @deprecated("Use maintainerScripts in RPM and RpmConstants.Pretrans instead.", "1.1.x")
```

Comment:
Just an FYI -

Deprecated warnings aren't really flagged well in sbt (Maybe we should open a ticket about that).

Additionally, we may want to issue the warning when someone sets these settings, i.e. something like:
```scala
val keyDeprecationWarnings: SettingKey[Seq[SettingKey]]
def warnIfSet(k: SettingKey[_]): Setting[_] = {
   warn ++= k.?.value match {
     case None => Seq()
     case Some(_) => Seq(k)
  }
}
```

This should be good enough for now, but we should probably check to see what the experience is upgraded.